lib/apiForSheets.js

```diff
@@ -101,21 +106,38 @@ module.exports.checkClaSheet = async function(auth) {
     auth: auth,
     spreadsheetId: spreadsheetId,
     range: 'Usernames!A:A',
-  }, function(err, response) {
+  }, async function(err, response) {
     if (err) {
       // eslint-disable-next-line no-console
       console.log('The API returned an error: ' + err);
       return;
     }
-    module.exports.generateOutput(response.values);
+
+    var pullRequestsPromiseObj = await context.github.pullRequests.getAll(
+      context.repo({per_page: 40}));
+
+    var arrayOfOpenPullRequests = pullRequestsPromiseObj.data;
+
+    for (var indexOfPullRequest in arrayOfOpenPullRequests) {
+      var pullRequestNumber = arrayOfOpenPullRequests[
+        indexOfPullRequest].number;
+      var pullRequestDetailsPromiseObj = await context.github.pullRequests.get(
+        context.repo({number: pullRequestNumber}));
+
+      var pullRequestDetails = pullRequestDetailsPromiseObj.data;
+
+      // eslint-disable-next-line no-console
```

Comment:
Do you need this still?

Reply:
We have no console log rule in eslintrc enabled. Therefore, we require this. Moreover, I wanted to keep logging to help us in debugging.